eth/ethutils/receipt.go

```diff
@@ -27,6 +27,8 @@ func MarshalReceipt(
 		if t.Protected() {
 			chainId = types.DeriveChainId(&t.V).ToBig()
 		}
+	case *types.DepositTx:
```

Comment:
Why was this case removed?  I believe that `GetChainID` will panic on deposit tx?

Reply:
This is a new file. Yes, deposit tx doesn't have a chain Id.